Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -33,9 +38,22 @@ public ShooterWheelSubsystem(
                 propManager);
         log.info("Creating");
         this.side = side;
+        flushToBoilerTargetSpeed = 
+                propManager.createPersistentProperty(side + " flush to boiler target speed", 3500);
     }
+    
     public RobotSide getSide(){
         return side;
     }
+    
+    public void setTargetSpeedForRange(ShooterRange range) {
+        switch (range) {
+        case FlushToBoiler:
```

Comment:
Sounds pretty good! I like the approach. For now it can be real simple, and eventually once we have a good formula for range -> wheel speeds, we can add that in transparently.